code/game/jobs/job/civilians.dm

```diff
@@ -150,7 +150,7 @@ You are also an expert when it comes to medication and treatment. If you do not
 	generate_entry_message(mob/living/carbon/human/H)
 		. = {"You are a civilian, and are not subject to follow military chain of command, but you do work for the USCM.
 You are tasked with researching and developing new medical treatments, helping your fellow doctors, and generally learning new things.
-Your role involves a lot of roleplaying, but you can perform the function of a regular doctor. Do not hand out things to marines without getting permission from your supervisor."}
+Your role involves some roleplaying and gimmickry, but you can perform the function of a regular doctor. You shouldn't hand out things to marines without getting permission from your supervisor."}
```

Comment:
Let's be real here, do we really need to tell them not to hand out things? They're basically the chemists of TG.